### PR TITLE
Writing Day - Convince Community and Blog CTA

### PR DESCRIPTION
**Pull request by rosewms**
Part 1, WIP, this is just the "Convince Your Community - Bring Your Project to Writing Day" page. Blog edits are happening now, PR will be updated shortly.

----
📚 Documentation preview 📚: https://writethedocs-www--2094.org.readthedocs.build/

* [Announcement Writing Day Call for Projects, direct link to preview](https://writethedocs-www--2094.org.readthedocs.build/conf/portland/2024/news/call-for-projects-writing-day/).
* [Convince your community - Writing Day, direct link to preview](https://writethedocs-www--2094.org.readthedocs.build/conf/portland/2024/convince-your-manager/#convince-your-community)

### CODE REVIEW

File: docs/conf/portland/2024/news/call-for-projects-writing-day.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Announcing the Writing Day Call for Projects
+============================================
+
+Hey folks! We were so thrilled with your response to Writing Day last year that we are launching 
+a `Call for Projects submission form <https://forms.gle/5WPN8MGCKzKqZBWU6>`_!
+
+For projects submitted by March 28, 2024, we have plans to highlight your information in our 
+pre-conference social media. We learned last year that projects that submit their information 
+pre-conference have higher attendance. In fact, perusing project details encourages documentarians 
+to join us earlier, stay longer, and check out multiple projects.
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/call-for-projects-writing-day.rst]:
```suggestion
For projects submitted by March 28, 2024, we'll highlight your project on our social media, as we learned last year that projects that submit their information before the conference have higher attendance. In fact, perusing project details encourages documentarians to join us earlier, stay longer, and check out multiple projects.
```

File: docs/conf/portland/2024/news/call-for-projects-writing-day.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+to learn new technologies, sharpen their skills, and add to their portfolios. The world of open 
+source has a seemingly endless number of wonderful open source projects. Documentarians want to write 
+the docs, your open source project docs.
+
+We are looking for community/OSS projects for our in-person Writing Day at the Portland conference on April 14.
+
+The `Call for Proposals submission form <https://forms.gle/5WPN8MGCKzKqZBWU6>`_ is open until *11:59 PM, March 28, 2024 PST*.
+
+(If you’d prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for our 
+Atlantic conference held online in a timezone as widely accessible as possible).
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/call-for-projects-writing-day.rst]:
```suggestion
(If you’d prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for our 
Atlantic conference held online in timezone selected for accessibilty.
```

**Review by plaindocs — APPROVED** [review]
Nice work @rosewms

**Comment by rosewms** [comment]
@katiejano made the requested changes from our Slack thread and combined the "Convince your..." pages to one page under the original "Convince your manager" context. Added info and a link to the Writing Day page as well. FYI @plaindocs ^

**Comment by plaindocs** [comment]
:shipit:

**Pull request merged**